src/pages/doi.org/[...doi].tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { gql, useQuery } from '@apollo/client'
 import { GetServerSideProps } from 'next'
+import { userAgentFromString } from 'next/server'
```

Comment:
Is this different to https://nextjs.org/docs/app/api-reference/functions/userAgent because we're on an older version?